Tests/UnitTests/Fatras/EventData/ParticleTests.cpp

```diff
@@ -49,7 +49,7 @@ BOOST_AUTO_TEST_CASE(Construct) {
 
 BOOST_AUTO_TEST_CASE(CorrectEnergy) {
   const auto pid = Barcode().setVertexPrimary(1).setParticle(42);
-  auto particle = Particle(pid, PdgParticle::eProton, 1_GeV, 1_e)
+  auto particle = Particle(pid, PdgParticle::eProton, 1_e, 1_GeV)
```

Comment:
Was this a bug?

Reply:
Yes, but it did not matter since both were numerical values `1`.